actions/preview/terraform/main.tf

```diff
@@ -1,19 +1,24 @@
 module "env" {
-  source = "github.com/broadinstitute/terraform-ap-modules.git//terra-preview-env?ref=terra-preview-env-0.0.0"
+  source = "github.com/broadinstitute/terraform-ap-modules.git//terra-env?ref=master"
```

Comment:
update to release tag?

Reply:
Yeah so I thought about this and having to manually update the tag here every time there's a new terra-env version seems like a lot? Just having it point to master and pick up whatever the latest version of the terra-env module seems fine?